metrics.go

```diff
@@ -47,6 +47,14 @@ var (
 	)
 )
 
+var pconfigCounter = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "pconfig_gets_total",
+		Help: "Counts the number of Get requests to pconfig",
+	},
+	[]string{"key"},
+)
+
```

Comment:
can you declare this in the same `var` block as all the other metrics?

Reply:
I had it separated because the first block is one set of metrics (for measuring performance on calls to our API) and this is for counting the times the `pconfig.Get` function is called. That being said, I've consolidated it because I'm fine with shortening the file and consolidating declarations.